backend/apps/iamstudent/templates/student_list_view.html

```diff
@@ -138,17 +133,17 @@
             </div>
             {% if enable_mail %}
             <button class="btn blue text-white btn-lg" onclick="sendall()">
-                Email an {{n}} Helfer schicken &raquo;
+                {% blocktrans %}Email an {{n}} Helfer schicken{% endblocktrans %} &raquo;
             </button>
             <button class="btn blue text-white btn-lg" onclick="send()">
-                Email an AUSGEWÄHLTE Studierende schicken &raquo;
+                {% blocktrans %}Email an AUSGEWÄHLTE Studierende schicken{% endblocktrans %} &raquo;
             </button>
             {% else %}
             <button class="btn blue text-white btn-lg" onclick="alertMaxMails()">
-                Email an {{n}} Helfer schicken &raquo;
+                {% blocktrans %}Email an {{n}} Helfer schicken{% endblocktrans %} &raquo;
```

Comment:
Muss `&raquo;` hier mit in den blocktrans?
```suggestion
                {% blocktrans %}Email an {{n}} Helfer schicken &raquo;{% endblocktrans %}
```